charts/workflow/values.yaml

```diff
@@ -165,6 +165,15 @@ registry-token-refresher:
 
 router:
   dhparam: ""
+  # Any custom router annotations(https://github.com/deis/router#annotations)
+  # which needs to applied can be specified as key values pair under this deployment_annotations key
```

Comment:
I would rewrite this slightly:
`which need to be applied can be specified as key-value pairs under "deployment_annotations"`